{
  "comments": [
    {
      "unresolved": false,
      "key": {
        "uuid": "f8bddfa2_7a93599c",
        "filename": "/PATCHSET_LEVEL",
        "patchSetId": 1
      },
      "lineNbr": 0,
      "author": {
        "id": 1037843
      },
      "writtenOn": "2024-05-13T11:21:13Z",
      "side": 1,
      "message": "big +1, this makes make the flow much easier to reason about",
      "revId": "efbfc4e9bc696538252b3a6ead01f365ea877768",
      "serverId": "d5d70762-12d0-45a1-890d-524b12d3f735"
    },
    {
      "unresolved": true,
      "key": {
        "uuid": "14d8fefd_af5adc63",
        "filename": "cmd/cue/cmd/common.go",
        "patchSetId": 1
      },
      "lineNbr": 732,
      "author": {
        "id": 1007275
      },
      "writtenOn": "2024-05-13T11:25:34Z",
      "side": 1,
      "message": "Maybe add a TODO here to remove calls to exitOnErr ?",
      "revId": "efbfc4e9bc696538252b3a6ead01f365ea877768",
      "serverId": "d5d70762-12d0-45a1-890d-524b12d3f735"
    },
    {
      "unresolved": true,
      "key": {
        "uuid": "79919bf9_13ce073a",
        "filename": "cmd/cue/cmd/vet.go",
        "patchSetId": 1
      },
      "lineNbr": 142,
      "author": {
        "id": 1037843
      },
      "writtenOn": "2024-05-13T11:25:05Z",
      "side": 1,
      "message": "I\u0027m wondering the reasoning behind not doing a `return iter.err()` here?",
      "range": {
        "startLine": 139,
        "startChar": 0,
        "endLine": 142,
        "endChar": 11
      },
      "revId": "efbfc4e9bc696538252b3a6ead01f365ea877768",
      "serverId": "d5d70762-12d0-45a1-890d-524b12d3f735"
    }
  ]
}